features/yield/YieldCalculator.tsx

```diff
@@ -24,10 +33,34 @@ const YieldCalculator = () => {
   const { empState } = EmpState.useContainer();
   const { expirationTimestamp } = empState;
   const { usdPrice } = Balancer.useContainer();
-
   const [tokenPrice, setTokenPrice] = useState<string | null>(
     usdPrice ? usdPrice.toString() : null
   );
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [selectedUserMode, setSelectedUserMode] = useState<string | null>(
```

Comment:
I think you should use a radio button instead of a menu, it reduces # of clicks

Reply:
sure & done :)